src/pluggy/_hooks.py

```diff
@@ -608,7 +606,7 @@ class _SubsetHookCaller(HookCaller):
         "_remove_plugins",
     )
 
-    def __init__(self, orig: HookCaller, remove_plugins: AbstractSet[_Plugin]) -> None:
+    def __init__(self, orig: HookCaller, remove_plugins: Set[_Plugin]) -> None:
```

Comment:
This one seems incorrect at first glance

Reply:
mypy or ruff (I cannot recally which one) complained and suggested using `Set` from `collections.abc` instead of deprecated `AbstractSet`.

Reply:
Yeah seems correct:

https://docs.python.org/3/library/typing.html#typing.AbstractSet

![image](https://github.com/user-attachments/assets/a0a27128-d1c2-48a6-87c7-1710befeb7e5)